control/mateqn.py

```diff
@@ -497,7 +497,27 @@ def care(A, B, Q, R=None, S=None, E=None, stabilizing=True):
     state space operations.  See :func:`~control.use_numpy_matrix`.
 
     """
+    if not stabilizing:
+        return care_slycot(A, B, Q, R, S, E, stabilizing)
+    else:
```

Comment:
This always favors the SciPy version over Slycot for `stabilizing=True`. I guess the SLICOT routine should be more efficient than calling 3 solvers for `X`, `G`, `L`.

Reply:
Found 7cf9630146a961c28c38817d48794525d22c6287 and #8. Has this ever been investigated thoroughly?

Reply:
Might be nicer to implement a `method` keyword as suggested in #255.

Reply:
I guess the significant number of uncovered lines as reported by coveralls comes from this change.

Reply:
One of my grad students actually encountered that bug with slycot’s `dare`. I’ll see if I can find the bug. Ok if we leave adding a `method` keyword to a different PR?

Reply:
I'm would be okay with deferring the method keyword to a later PR.

But I don't like leaving code in the repo which is essentially untested or disabled.